### PR TITLE
Pete Cuppernull - Homework 1

### PR DESCRIPTION
**Pull request by pcuppernull**
I saved the original README file to the dropbox folder I am using for this class. I access it within R by clicking the upper right Git tab and selecting the README file. From there, I made my changes, committed them, and pushed them back up to Github.

I have some experience in R but none with Git/Github, so I found working with the latter to be more challenging. For the R part, I referenced the "Mastering Markdown" document for the lists, links, etc - coding these was a good reminder that just one space or punctuation mark will make all the difference in whether or not something works! 

For Git, I am on a Mac so I didn't have many issues getting that up and running. I asked for help along the way to confirm I was doing my pull properly.

### CODE REVIEW

**Comment by yiranouyang** [comment]
Topic | Excellent | Satisfactory | Needs Work
-- | -- | -- | --
Coding style | x |   |  
Coding strategy | x |   |  
Presentation | x |   |  
Achievement, creativity | x |   |  
Ease of access | x |   |  

## Comments
- Nice job of using the markdown language properly (headings, bold, image, link, and bullet points)!

- In the future, try to include only one version of the homework in the repository instead of having both **README.md** and **README.html**. This will help readers understand which version is your final version.

- Your reflection of the work process was helpful for me because I also have previous experience coding in R but am new to Github/Git. Overall, you did quality work. Thank you for sharing!

File: README.md [hunk 1]
```diff
@@ -1,2 +1,24 @@
-# hw01
```
Comment (jfan3) [inline README.md:1]: Nice!

File: README.md [hunk 2]
```diff
@@ -1,2 +1,24 @@
...
-Repository for homework 01
+# Pete Cuppernull 
+
+## Computing for the Social Sciences
```
Comment (jfan3) [inline README.md:3]: You can use the headers to sectionalize the layout.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,24 @@
...
+
+## Homework Assignment 1
+
+### **Biography**
+
+My name is **Pete Cuppernull** and I am a first year PhD student in the political science department. I'm studying international relations and quantitative methods and plan to complete the CSS certificate program. I'm interested in studying humanitarian interventions and specifically how states choose to pursue different types of interventions abroad. 
+
+Before coming to UChicago, I lived in New York City. When I first moved to New York, I worked in the financial technology industry where I researched corporate governance and shareholder activism. I ultimately transitioned to a product development role at my firm where I designed a few data collection applications and gained experience with SQL and Python. I left the finance world in 2017 to pursue my other passion in music, and worked as a music producer in the city for a couple of years. I've built an artist project with my older brother producing deep house and techno music, and I have worked as a sound engineer for pop artists as well. A handful of artists I've worked for are: 
+
+* The Chainsmokers
```
Comment (jfan3) [inline README.md:13]: Cool!

File: README.md [hunk 4]
```diff
@@ -1,2 +1,24 @@
...
+* Billie Eilish
+
+After exploring these other interests, I decided to return to school to pursue my education in international relations and peace studies. I'm excited to continue to build my computational skills in this class and look forward to a great quarter!
+
+Here is a photo of myself. 
+
+![Here is a photo of myself.](photo.jpg)
+
+
+[Here is a link to the UChicago Homepage](https://www.uchicago.edu/)
```
Comment (jfan3) [inline README.md:24]: You can do it like `[Here] is a link to the UChicago Homepage(https://www.uchicago.edu/)`. This link seems a bit out of place.

**Review by jfan3 — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |x           |              |            |
| **Coding strategy**         |x           |              |            |
| **Presentation**            |            |x              |            |
| **Achievement, creativity** |           |               |x            |
| **Ease of access**          |x           |              |            |

Remarks:
* Nice work overall. 
* You can explore the different fonts (italics, bold, underscore, etc) a little more. 
* The layout can be improved.

**Review by bensoltoff — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |      x     |              |            |
| **Coding strategy**         |    x       |              |            |
| **Presentation**            |      x     |              |            |
| **Achievement, creativity** |    x       |              |            |
| **Ease of access**          |     x      |              |            |

Remarks:

* Great addition of a photo to the repo. If it isn't already hosted online, you can just put it in the repo directly.
* [Some files you don't need to commit](https://cfss.uchicago.edu/faq/homework-guidelines/#which-files-to-commit). We will learn more about this as the course goes on, but for example the preview file `README.html` doesn't need to be committed. This is because you only need to view `README.md` on GitHub - all you see when you click on the [HTML file](https://github.com/uc-cfss/hw01/blob/aa8d31f08b72f591ada21c4d2d7dae162f65a69a/README.html) is a bunch of source code.

**Pull request closed without merging**